image_data_guide/01_download_data.ipynb

```diff
@@ -0,0 +1,546 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Downloading and Combining Datasets (Part 1/4)\n",
+    "**Download** | Structure | Preprocess | Train"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import pickle\n",
+    "import shutil\n",
+    "import urllib\n",
+    "import pathlib\n",
+    "import tarfile\n",
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "import matplotlib.pyplot as plt\n",
+    "from imageio import imread, imwrite\n",
+    "from joblib import Parallel, delayed, parallel_backend"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<pre>\n",
+    "</pre>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
Let's add some info about the dataset before downloading it.

Reply:
Added a short intro about the COCO and CIFAR datasets